### PR TITLE
Add account management UI info to user guide

### PR DESCRIPTION
**Pull request by kprosise**
Added two pages to document account management tasks, covering billing and access, to assist with common customer questions. Team based access will be updated in a future commit to avoid creep. The pages link to each other, which is why I made the decision to bundle them in one commit.

An additional change was to move the account management section to the top of the user guide, based on feedback.

Future changes will likely be needed as questions arise, and these pages are not meant to be comprehensive at this time.

QA steps taken: quick pass with vale linter, site rendered as html and visually inspected. Ran link-check. No issues were detected.

This commit applies to FFTK 1305 and FFTK 1611

Signed-off-by: Katrina Prosise <katrina.prosise@foundries.io>

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [x] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for a9c3cd70ad58e6c4abc3862640e76b39b685cfd6 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1668/docs/artifacts/html/index.html

File: source/user-guide/account-management/account-roles.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+The remaining two each get a subset of the Owner permissions.
+
+* **Admin**: Add and update members, and :ref:`manage teams <ref-team-based-access>`.
+* **Accounting** limited to managing the :ref:`Factory plan <ref-subscription-and-billing>`.
+
+.. tip::
+    No limits exist on how many members can share a role.
+    For example, more than one member can have the owner role.
+    However,a single member has only one role at a time.
```
Comment (vanmaegima) [inline source/user-guide/account-management/account-roles.rst:27]:
```suggestion
    However, a single member has only one role at a time.
```

File: source/user-guide/account-management/account-roles.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+.. csv-table:: Roles and Permission Summary
```
Comment (vanmaegima) [inline source/user-guide/account-management/account-roles.rst:29]: This table shows an addition blank column at the end

File: source/user-guide/account-management/account-roles.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+.. figure:: /_static/userguide/account-management/invite-members.png
+   :align: center
+   :alt: sending member invites
+
+.. tip::
+   When inviting members, you can also select any teams to assign them automatically.
+
+On creating an account and accepting the invitation, members have no account management permissions.
+
+From the :guilabel:`Member` tab, the owner selects members from the table, and clicks on :guilabel:`role...`,
```
Comment (vanmaegima) [inline source/user-guide/account-management/account-roles.rst:53]:
```suggestion
From the :guilabel:`Member` tab, the owner selects members from the table, and clicks on :guilabel:`Role...`,
```

File: source/user-guide/account-management/subscription-and-billing.rst [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+To continue after the trial has expired,
+go to your Factory page's plan tab and click on "Upgrade" under available plans. 
+From here enter the required information:
+
+.. figure:: /_static/userguide/account-management/upgrade-plan.png
+   :width: 600
+   :align: center
+   :alt: add billing details for upgrading plan
+
+For using other forms of payment, please `contact us <https://foundriesio.atlassian.net/servicedesk/customer/portal/1/group/1/create/2>`_
```
Comment (vanmaegima) [inline source/user-guide/account-management/subscription-and-billing.rst:42]:
```suggestion
For using other forms of payment, please `contact us <https://foundriesio.atlassian.net/servicedesk/customer/portal/1/group/1/create/2>`_.
```

File: source/user-guide/account-management/account-roles.rst [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+* **Member**: Default when adding new users, with no permissions.
+* **Owner**: The user account that created the factory has the initial "Owner" role, with full permission to:
+  
+  - add and update members
+  - manage teams
+  - manage the Factory subscription plan.
+
+The remaining two each get a subset of the Owner permissions.
+
+* **Admin**: Add and update members, and :ref:`manage teams <ref-team-based-access>`.
```
Comment (doanac) [inline source/user-guide/account-management/account-roles.rst:21]: Admins and owners can also manage the Factory's PKI as well as create Waves

**Comment by vanmaegima** [comment]
There's some other information I think we can explore, like each member can manage their devices, and the factory owner/admin can manage all devices, also who has source code access and can push changes.
But I don't know if you'd like to address this in a different section of the docs (not related to account/billing).

File: source/user-guide/account-management/account-roles.rst [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+* **Accounting** limited to managing the :ref:`Factory plan <ref-subscription-and-billing>`.
+
+.. tip::
+    No limits exist on how many members can share a role.
+    For example, more than one member can have the owner role.
```
Comment (angolini) [inline source/user-guide/account-management/account-roles.rst:26]:
```suggestion
    For example, more than one member can have the **Owner** role.
```

File: source/user-guide/account-management/account-roles.rst [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    However,a single member has only one role at a time.
+
+.. csv-table:: Roles and Permission Summary
+   :header: "Role", "Edit Members", "Manage Teams", "Subscription/Billing", 
+
+   "Owner", "X", "X", "X"
+   "Admin", "X", "X",
+   "Accounting", , "X"
+   "Member", , ,
```
Comment (angolini) [inline source/user-guide/account-management/account-roles.rst:35]: I would start with the table, and then the text

File: source/user-guide/account-management/subscription-and-billing.rst [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+Managing your Subscription
+==========================
+
+.. hint::
+   FoundriesFactory is a DevSecOps platform subscription service.
+   Per month or annual plan choices,
+   both provide access to all FoundriesFactory features and support.
+
+Subscriptions generally apply per Factory;
```
Comment (angolini) [inline source/user-guide/account-management/subscription-and-billing.rst:11]: what does "generally" means here? Would be better to just drop the word?

File: source/user-guide/account-management/subscription-and-billing.rst [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+   :align: center
+   :alt: add billing details for upgrading plan
+
+For using other forms of payment, please `contact us <https://foundriesio.atlassian.net/servicedesk/customer/portal/1/group/1/create/2>`_
+
+Payment and Billing
+--------------------
+
+To make changes —such as payment method or autorenewal— you must have an Owner or Accounting role.
+Changes can be made through the plan tab.
```
Comment (angolini) [inline source/user-guide/account-management/subscription-and-billing.rst:48]:
```suggestion
Changes can be made through the :guilabel:`Plan` tab.
```

File: source/user-guide/account-management/subscription-and-billing.rst [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+   :width: 600
+   :align: center
+   :alt: add billing details for upgrading plan
+
+For using other forms of payment, please `contact us <https://foundriesio.atlassian.net/servicedesk/customer/portal/1/group/1/create/2>`_
+
+Payment and Billing
+--------------------
+
+To make changes —such as payment method or autorenewal— you must have an Owner or Accounting role.
```
Comment (angolini) [inline source/user-guide/account-management/subscription-and-billing.rst:47]:
```suggestion
To make changes —such as payment method or autorenewal— you must have an **Owner** or **Accounting** role.
```

File: source/user-guide/account-management/account-roles.rst [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+A role consists of set permission levels for **account management** that a *member* of a FoundriesFactory gets assigned.
+
+* **Member**: Default when adding new users, with no permissions.
+* **Owner**: The user account that created the factory has the initial "Owner" role, with full permission to:
+  
+  - add and update members
+  - manage teams
+  - manage the Factory subscription plan.
+
+The remaining two each get a subset of the Owner permissions.
```
Comment (angolini) [inline source/user-guide/account-management/account-roles.rst:19]:
```suggestion
The remaining two each get a subset of the **Owner** permissions.
```

**Review by angolini — APPROVED** [review]

File: source/user-guide/account-management/subscription-and-billing.rst [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+   Per month or annual plan choices,
+   both provide access to all FoundriesFactory features and support.
+
+Subscriptions generally apply per Factory;
+if you have *Factories*, you will manage them separately.
+
+Changing your Plan
+------------------
+
+From your Factory's page, navigate to the :guilabel:`Plan` tab.
```
Comment (MiloCasagrande) [inline source/user-guide/account-management/subscription-and-billing.rst:17]: Just a heads-up here: I plan to move the `Plan` tab inside the `Settings` one in the future. I have a few changes already lined up in the UI to prepare for that, but I don't know when I'll start moving the `Plan` tab. There will be a time where we will have the `Plan` tab and the same actions inside the `Settings` one.

I'll let you know in advance to prepare for docs update.

File: source/user-guide/account-management/account-roles.rst [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+
+Factory Account Roles
+=====================
+
+ 
+Managing and utilizing a Factory  various *roles*, regardless the number of members;
+
+
+A role consists of set permission levels for **account management** that a *member* of a FoundriesFactory gets assigned.
+
```
Comment (kprosise) [inline source/user-guide/account-management/account-roles.rst:11]:
```suggestion
There are four roles available. The two core roles being:

```

File: source/user-guide/account-management/account-roles.rst (hunk 5 above)
Comment (kprosise) [inline source/user-guide/account-management/account-roles.rst:21]:
```suggestion
* **Admin**: Add and update members, :ref:`manage teams <ref-team-based-access>`, manage PKI, and create Waves.
```

File: source/user-guide/account-management/account-roles.rst (hunk 7 above)
Comment (kprosise) [reply source/user-guide/account-management/account-roles.rst:35]: Good idea, especially for anyone who just wants to refresh or check their understanding.

File: source/user-guide/account-management/account-roles.rst [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+    However,a single member has only one role at a time.
+
+.. csv-table:: Roles and Permission Summary
+   :header: "Role", "Edit Members", "Manage Teams", "Subscription/Billing", 
+
+   "Owner", "X", "X", "X"
+   "Admin", "X", "X",
+   "Accounting", , "X"
+   "Member", , ,
+
```
Comment (kprosise) [inline source/user-guide/account-management/account-roles.rst:36]:
```suggestion
```

File: source/user-guide/account-management/account-roles.rst [hunk 15]
```diff
@@ -0,0 +1,64 @@
+.. _ref-account-roles:
+
+Factory Account Roles
+=====================
+
+
```
Comment (kprosise) [inline source/user-guide/account-management/account-roles.rst:6]:
```suggestion
 .. csv-table:: Roles and Permission Summary
   :header: "Role", "Edit Members", "Manage Teams", "Subscription/Billing", "Manage PKI", "Create Waves"

   "Owner", "X", "X", "X", "X", "X"
   "Admin", "X", "X", ,"X", "X"
   "Accounting", , "X", , ,
   "Member", , , , ,
```
Summary table should be at top. PKI and Wave info should be included.

File: source/user-guide/account-management/subscription-and-billing.rst (hunk 8 above)
Comment (kprosise) [inline source/user-guide/account-management/subscription-and-billing.rst:11]:
```suggestion
Subscriptions apply per Factory;
```

**Comment by doanac** [comment]
Docs for 41850b9ab048c48b6683bc7b2c246d90ed10c9f3 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1674/docs/artifacts/html/index.html

File: source/user-guide/account-management/account-roles.rst [hunk 16]
```diff
@@ -0,0 +1,65 @@
...
+ .. csv-table:: Roles and Permission Summary
+   :header: "Role", "Edit Members", "Manage Teams", "Subscription/Billing", "Manage PKI", "Create Waves"
+
+   "Owner", "X", "X", "X", "X", "X"
+   "Admin", "X", "X", ,"X", "X"
+   "Accounting", , "X", , ,
+   "Member", , , , ,
+   
+Managing and utilizing a Factory various *roles*, regardless of the number of members;
```
Comment (kprosise) [inline source/user-guide/account-management/account-roles.rst:14]:
```suggestion
```
Line is not needed and seemed out of place

File: source/user-guide/account-management/account-roles.rst [hunk 17]
```diff
@@ -0,0 +1,65 @@
...
+
+
+A role consists of set permission levels for **account management** that a *member* of a FoundriesFactory gets assigned.
```
Comment (kprosise) [inline source/user-guide/account-management/account-roles.rst:17]:
```suggestion
```
Moving to above table as introductory sentence. removing extra whitespace

File: source/user-guide/account-management/account-roles.rst [hunk 18]
```diff
@@ -0,0 +1,65 @@
+.. _ref-account-roles:
+
+Factory Account Roles
+=====================
+
```
Comment (kprosise) [inline source/user-guide/account-management/account-roles.rst:5]:
```suggestion

A role consists of set permission levels for **account management** that a *member* of a FoundriesFactory gets assigned.

```

**Review by kprosise — COMMENTED** [review]
Further clean up after changes introduced during review.

**Comment by doanac** [comment]
Docs for 2f6c150df810cd42bc4242315620e1d487492df2 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1685/docs/artifacts/html/index.html

**Pull request merged**